src/server/controllers/verification/session-state/input-files.session-state.paths.yaml

```diff
@@ -39,6 +38,12 @@ paths:
             format: uri
           description: Remote file URL
           required: false
+        - name: dryrun
+          in: query
+          schema:
+            type: boolean
```

Comment:
Shall we declare a `default`[1] value for the parameter, _i.e._ `false`? This is provide the user with this information. 

[1] _Default Parameter Values_ in https://swagger.io/docs/specification/describing-parameters/